src/cincinnati/mod.rs

```diff
@@ -35,6 +41,11 @@ lazy_static::lazy_static! {
         "zincati_cincinnati_graph_edges_count",
         "Number of edges in Cincinnati update graph."
     )).unwrap();
+    static ref RELEASE_IS_DEADEND: IntGaugeVec = register_int_gauge_vec!(
+        "zincati_cincinnati_release_is_deadend",
+        "Whether current release is a dead-end.",
+        &["version"]
```

Comment:
Don't we already expose the OS version via `zincati_identity_os_info` ? This could just be a single `IntGauge` no?

Reply:
Ack, done.